frontend/src/cljs/imas_seamap/db.cljs

```diff
@@ -45,6 +45,7 @@
    :display         {:help-overlay    false
                      :welcome-overlay false
                      :seamap-drawer   false
+                     :drawer-panels   []
```

Comment:
Let's get this added to the spec (also the drawer, noting other feedback about naming too).
Also worth thinking about how we communicate how this list is used (is the current panel first or last in the last?  I see it's "last" from the events below, but eg calling it drawer-panel-stack, and I think there's push/pop functions even which might be clearer?)

Reply:
I was half right: there's a [pop](https://clojuredocs.org/clojure.core/pop), but no push!  I guess that's what conj does.  @olivergeorge can comment.

Reply:
Okay, so I've renamed the events to push and pop, and drawer-panels to drawer-panel-stack to better reflect their nature as a stack.